netjsonconfig/schema.py

```diff
@@ -443,23 +443,38 @@
                 }
             }
         },
+        "encryption_mfp_property": {
+            "properties": {
+                "ieee80211w": {
+                    "type": "string",
```

Comment:
we should be able to add a title property here which would replace "ieee80211w" in the UI, something like `"title": "Management frame protection",`, what do you think?

Reply:
Thank you for your review !  It is more clear for user. I fixed it. I used lowercase letters because other items are using lowercase letters.